packages/data-models/constants.ts

```diff
@@ -48,6 +50,14 @@ const APP_LANGUAGES = {
   default: "gb_en",
 };
 
+/** Specifies the skins available to the current app build.
+ * "default": the skin used by default for a given build
+ * "available": an array of additional skins that are available to the current build */
+const APP_SKINS: { default: IAppSkin; available: IAppSkin[] } = {
+  default: { name: "default" },
```

Comment:
nit(blocking)
The default should probably just be a string containing the name instead of a partial skin. The skin will then be looked up from the list of available, applying the changes if found (possibly just logging error if not). Otherwise it could be tempting to pass a full skin into the default property